ui-tests/src/test/java/org/openmrs/reference/page/PatientCaptureVitalsPage.java

```diff
@@ -127,4 +127,36 @@ public void checkIfRightPatient() {
             confirmButton2.click();
         }
     }
+    
+    public void clearPatientHeight() {
+        findElement(HEIGHT_FIELD).clear();
+    }
+    
+    public void clearPatientWeight() {
+        findElement(WEIGHT_FIELD).clear();
+    }
+    
+    public void clearPatientTemperature() {
+        findElement(TEMPERATURE_FIELD).clear();
+    }
+
+    public void clearPatientPulse() {
+        findElement(PULSE_FIELD).clear();
+    }
+
+    public void clearPatientRespiratoryRate() {
+        findElement(RESPIRATORY_FIELD).clear();
+    }
+
+    public void clearPatientBloodPressure1() {
+        findElement(BLOOD_PRESSURE_FIELD_1).clear();
+    }
+
+    public void clearPatientBloodPressure2() {
+        findElement(BLOOD_PRESSURE_FIELD_2).clear();
+    }
+
+    public void clearPatientBloodOxygenSaturation() {
+        findElement(BLOOD_OXYGEN_SATURATION_FIELD).clear();
+    }
 }
```

Comment:
The elements look ups are already defined at code `LN-13` to `LN-20` within the same file?